### DISCUSSION TITLE
怎么去保存识别输出的结果呢？

### DISCUSSION THREAD

**Discussion by STSTERANDMOMO** [PaddlePaddle/PaddleOCR#12699] [Q&A]
!paddleocr --image_dir ./testset/testdata/word_0.jpg --use_angle_cls true --use_gpu true --det false --output ./output 
我在服务器上使用了以上指令来进行文本识别，输出结果是一个list，怎么把这个结果保存下来呢，我应该怎么进行修改呢

**Comment by Leon-Cheung-CQ** [comment]
用脚本跑吧，https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_ch/quickstart.md#21
